src/mca/schizo/ompi/help-schizo-ompi.txt

```diff
@@ -36,30 +36,29 @@ Initiate an instance of the PMIx Reference RTE (PRRTE) DVM
    --debug-daemons-file              Enable debugging of any PRTE daemons used by this application, storing
                                      their verbose output in files
    --display <arg0>                  Comma-delimited list of options for displaying information about the
-                                     allocation and job. Allowed values: allocation, bind, map, map-devel,
-                                     topo
+                                     allocation and job. Allowed values for <arg0>: allocation, bind,
```

Comment:
What about putting <CR> after `<arg0>:` and putting all of the values on the next line together?  This is a preferred line splitting if we have to split the line.